### PR TITLE
CDAP-3542 Unit test case for the failure in the Workflow fork.

### PR DESCRIPTION
**Pull request by sagarkapare**
JIRA: https://issues.cask.co/browse/CDAP-3542
Build: http://builds.cask.co/browse/CDAP-DUT2914-1

### CODE REVIEW

**Comment by sagarkapare** [comment]
@anwar6953 Please take a look. Thanks!

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java [hunk 1]
```diff
@@ -1325,4 +1327,31 @@ public void testWorkflowTokenPut() throws Exception {
     workflowProgramRuns = getProgramRuns(sparkId, ProgramRunStatus.COMPLETED.name());
     Assert.assertEquals(1, workflowProgramRuns.size());
   }
+
+  @Test
+  public void testWorkflowForkFailure() throws Exception {
```
Comment (anwar6953) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java]: I think you should add a comment, briefly describing the scenario you are testing.
Other than that, LGTM.

Comment (sagarkapare) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java]: Updated the description.

Comment (anwar6953) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java]: LGTM

**Comment by sagarkapare** [comment]
Build passed. Merging.

**Pull request merged**